src/components/practitioner.ts

```diff
@@ -1,5 +1,5 @@
 
-export default class Practitioner {
+export class Practitioner {
```

Comment:
For future reference, you can still `export default` and also `export named` stuff below. You'd import like so:
```js
import Practitioner, { instanceOfPhone, instanceOfAddress } from 'blah'
```

But that's not a blocker to getting this merged! ✔️